pymeasure/instruments/keithley/keithley6517b.py

```diff
@@ -0,0 +1,343 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2019 PyMeasure Developers
```

Comment:
Please add yourself to the authors.
https://github.com/ralph-group/pymeasure/blob/master/AUTHORS.txt
https://github.com/ralph-group/pymeasure/blob/master/docs/about/authors.rst

Reply:
done